.github/ISSUE_TEMPLATE/add-reference.md

```diff
@@ -17,27 +17,29 @@ Requesting to add the reference below:
 
 ---
 
+<!-- prettier-ignore-start -->
+
 > ### 📚 **[PAPER_TITLE](PAPER_URL)**
 >
 > - 📅 **Publication Date:** MONTH DD, YYYY
 > - 📘 **Journal:** JOURNAL_NAME
 > - 🔗 **DOI:** [DOI_NUMBER](DOI_URL)
 >
-> [🔍 EG_MORE_DETAILS](URL_1) > [💻 EG_LINK_2_CODE](URL_2)
+> [🔍 EG_MORE_DETAILS](URL_1)<br> > [💻 EG_LINK_2_CODE](URL_2)<br>
```

Comment:
This is virtually the same as 
```md
> [🔍 EG_MORE_DETAILS](URL_1)
>
> [💻 EG_LINK_2_CODE](URL_2)
```
But not using `<br>` reduces the risk of messing things up with copy-pasting.

Reply:
Seams to be different in the preview and actual application, so just ignore it!